guides/source/association_basics.md

```diff
@@ -110,13 +110,37 @@ class CreateBooks < ActiveRecord::Migration[5.0]
 
     create_table :books do |t|
       t.belongs_to :author, index: true
-      t.datetime :published_at
+      t.string :name
```

Comment:
I am not sure I understand this one. The other suggestions look good. I will work on this and resubmit.